src/core/scene/a-scene.js

```diff
@@ -251,7 +251,17 @@ module.exports.AScene = registerElement('a-scene', {
           vrDisplay = utils.device.getVRDisplay();
           vrManager.setDevice(vrDisplay);
           vrManager.enabled = true;
-          vrManager.setPoseTarget(this.camera.el.object3D);
+
+          if (this.camera) {
+            vrManager.setPoseTarget(this.camera.el.object3D);
+          }
+          // If the camera was not available, we skip setPoseTarget.
+          // We expect it to be set as a result of the camera-set-active event at a later point
+          // (usually shortly after entering VR).
+          this.addEventListener('camera-set-active', function () {
```

Comment:
This event listener will be added no matter what. Maybe add this to `createdCallback`, so it's only ever added once, and then you don't even need the `this.camera` check perhaps?

Reply:
That might have consequences...at least we need an `else` with a one-time event listener.

Reply:
I figured we'd want it to always listen continuously, since a dev might decide to swap the active camera at a later point.

Reply:
If you enter and leave VR, this will add multiple listeners

Reply:
Ah, duh. Submitted a fix.